src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -90,18 +95,19 @@ public async Task BulkDocs()
                 if (command.IsClusterTransaction)
                 {
                     ValidateCommandForClusterWideTransaction(command, disableAtomicDocumentWrites);
+                    var raftRequestId = GetRaftRequestIdFromQuery();
 
-                    using (Database.ClusterTransactionWaiter.CreateTask(out var taskId))
+                    using (Database.ClusterTransactionWaiter.CreateTask(id: raftRequestId, out _))
```

Comment:
It seems we don't use the `out` anywhere, so let's remove it.

Reply:
Removed